EKF/terrain_estimator.cpp

```diff
@@ -283,21 +288,20 @@ void Ekf::fuseFlowForTerrain()
 	}
 }
 
-// return true if the terrain height estimate is valid
-bool Ekf::get_terrain_valid()
+bool Ekf::isTerrainEstimateValid()
 {
 	return _hagl_valid;
 }
 
-// determine terrain validity
-void Ekf::update_terrain_valid()
+void Ekf::updateTerrainValidity()
 {
 	// we have been fusing range finder measurements in the last 5 seconds
-	bool recent_range_fusion = (_time_last_imu - _time_last_hagl_fuse) < 5*1000*1000;
```

Comment:
@bresch If I remember correctly this was done to avoid a cast from double to unit64, since 5e6 is stored as a double.

Reply:
@bresch I think we should use time literals as we used them in the firmware. Here it would be 5 * 1_s